config/run-tests.sh

```diff
@@ -6,7 +6,7 @@ echo "$LABS"
 
 for lab in $LABS; do
 	echo "Running tests for lab #${lab}"
-
+	export PYTHONPATH=./lab_"${lab}"
```

Comment:
It should be also in the lint running tests script, at the top.

Reply:
done